packs/spell-effects/spell-effect-courageous-anthem.json

```
@@ -53,6 +53,11 @@
                     }
                 ],
                 "selector": "strike-damage"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "composition-signature:origin:signature:{item|origin.signature}"
```

```suggestion
                "option": "courageous-anthem:origin:signature:{item|origin.signature}"
```
We need it to have the name of the composition in there otherwise someone with two compositions on them would have both benefit from fortissimo. This will preclude 3pp but there's not much to be done for it.